models/modelling/commissioning/mh_currencies/int_mhsds_contact_currency.sql

```
@@ -0,0 +1,159 @@
+    ) = 1
+)
+
+, categorised as (
+    select
+        c.uniq_care_cont_id
+        , c.uniq_serv_req_id
+        , c.person_id
+        , b.sk_patient_id
+        , c.org_id_prov
+        , c.care_cont_date
+        , c.age_care_cont_date
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)